docs/sql/data_types/text.md

```diff
@@ -28,56 +28,14 @@ CREATE TABLE strings (
 
 The `VARCHAR` field allows storage of Unicode characters. Internally, the data is encoded as UTF-8.
 
-## Formatting Strings
+## Text type values
 
-Strings in DuckDB are surrounded by single quote (apostrophe) characters (`'`):
+Values of the text type are character strings, also known as string values or simply strings. At runtime, string values are constructed in one of the following ways:
 
-```sql
```

Comment:
@rpbouman why was this section and the "Strings with Special Characters" section below (explaining `chr`)  deleted?